### PR TITLE
#12 - IT with auth

### LINKED ISSUE #12: Add basic authorization
Let's add basic authorization to the repo, do not forget about tests (ITs included).

### PR DESCRIPTION
**Pull request by olenagerasimova**
Closes #12 
Added IT with auth.

### CODE REVIEW

File: src/test/java/com/artipie/debian/DebianAuthSliceITCase.java [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+        con.addRequestProperty(
+            "Authorization",
+            String.format(
+                "Basic %s",
+                new String(
+                    Base64.encodeBase64(DebianAuthSliceITCase.AUTH.getBytes(StandardCharsets.UTF_8))
+                )
+            )
+        );
+        con.setRequestMethod("PUT");
```
Comment (olegmoz) [inline src/test/java/com/artipie/debian/DebianAuthSliceITCase.java:128]: @olenagerasimova I'd suggest t extract a method for sending HTTP request with auth header, as it is repeated multiple times in the class

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see one comment

**Comment by olenagerasimova** [comment]
@olegmoz thanks, corrected

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, good to merge

**Pull request merged**